lib/ddtrace/sampler.rb

```diff
@@ -0,0 +1,48 @@
+
+module Datadog
+  # \Sampler performs client-side trace sampling.
+  class Sampler
+    def sample(span); end
```

Comment:
do we need this class as a kind of interface? I assume users can do `Datadog::Sampler.new().sample(span)` (which is wrong) so I think it should raise an error like:
```ruby
class Sampler
  def sample(span);
    raise 'You have to implement this method' # or whatever
  end
end
```

Obviously if it's Ruby-idiomatic.